modules/accredited_representative_portal/spec/sidekiq/accredited_representative_portal/email_personalisations_spec.rb

```diff
@@ -29,9 +29,57 @@
     context 'when type is declined' do
       let(:type) { 'declined' }
 
-      it 'returns a hash with the first name' do
-        expected_hash = { 'first_name' => notification.claimant_hash['name']['first'] }
-        expect(described_class::Declined.new(notification).generate).to eq(expected_hash)
+      before do
+        resolution = double('resolution', resolving: double('resolving', declination_reason))
+        allow(notification.power_of_attorney_request).to receive(:resolution).and_return(resolution)
+      end
```

Comment:
This is causing the tests to fail because `declination_reason` is being passed as a second positional argument instead of a method stub.

To fix it, `declination_reason` needs to be passed as a key-value pair, like this:

```
      before do
        resolution = double('resolution', resolving: double('resolving', declination_reason: declination_reason))
        allow(notification.power_of_attorney_request).to receive(:resolution).and_return(resolution)
      end
```